libs/core/src/global/styles/tokens/_line-height.scss

```diff
@@ -4,7 +4,7 @@
   --pine-line-height-md: 24px;
   --pine-line-height-lg: 28px;
   --pine-line-height-xl: 32px;
-  --pine-line-height-2xl: 40px;
+  --pine-line-height-xxl: 40px;
   --pine-line-height-3xl: 48px;
```

Comment:
Not related to these updates, but we should decide if we want to use numbers or letters for these variations. 3xl and 4xl as examples.

Reply:
I wonder if 3xl and 4xl is an exception and makes it more legible? Thoughts @Kajabi/dss-devs?

## With numbers
```
:root {
  --pine-line-height-xxl: 40px;
  --pine-line-height-3xl: 48px;
  --pine-line-height-4xl: 60px;
}
```

## With letters
```
:root {
  --pine-line-height-xxl: 40px;
  --pine-line-height-xxxl: 48px;
  --pine-line-height-xxxxl: 60px;
}
```

Reply:
I like the number variation. I don't have time to count the number of `x's` LOL.

Reply:
Fair enough and once we get into more semantic tokens these will change anyway.